omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -22,6 +22,10 @@
 
 	<!-- Add here beans related to the web context -->
 
-	<context:component-scan base-package="org.openmrs.module.appointments.web" />
+	<bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">
+		<property name="moduleId" value="appointments"/>
+	</bean>
+
+	<context:component-scan base-package="org.openmrs.module.appointments.web.controller" />

```

Comment:
There are Spring beans in this module defined in the `org.openmrs.module.appointments.web` package that are not part of the `controller` sub-package.